guide/onboarding/creating-a-new-wallet.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: guide
+title: Creating a new a wallet
+description: Steps to take when creating a new wallet
+parent: Onboarding
+nav_order: 2
+permalink: /guide/onboarding/creating-a-new-wallet/
+main_classes: -no-top-padding
+---
+
+
+# What is a bitcoin wallet?
+
+First, bitcoin [wallets](https://bitcoin.design/guide/glossary/#wallet) are not analogous to their namesakes. Physical wallets are places to store physical currency, credit cards, IDs, and so on. Bitcoin wallets don't actually store bitcoin.
+
+Instead, a bitcoin wallet contains the "private keys" to your bitcoin, the bitcoin is located on a network that contains a communally sustained public ledger with a record of everyone's transactions and balances (so think of bitcoin wallets more like keychains). Like regular keys on a keychain, these keys can be copied. For example, if you gave someone a copy of your house keys, they could get into your home. Likewise, if someone has a copy of your private key, they can access your funds. This makes private key management one of the most important considerations of any bitcoin owner.
```

Comment:
> communally sustained public ledger

As a non-native speaker, I don't really find it easy to understand this.

Reply:
Yeah I don't love this, in fact, the term "public ledger" feels very antiquated and medieval to me. How about a spreadsheet?